src/css/map-view.css

```diff
@@ -1665,3 +1665,99 @@ other class: .ui-slider-range */
   }
 }
 /** END - Expansion panel View. */
+
+/*****************************************************************************************
+ *
+ * Legend
+ *
+ * The full legend that overlays the map
+ *
+ */
+
+.legend-container {
+  z-index: var(--map-z-index-base);
+  position: absolute;
+  left: 1rem;
+  bottom: 1rem;
+  border-radius: var(--map-border-radius-big);
+  background: var(--portal-col-neutral-00, var(--map-col-bkg__deprecate));
+  width: 6.875rem;
+
+  .legend-container__header {
+    border-radius: var(--map-border-radius-big);
+    background: var(
+      --portal-col-primary-2,
+      var(--portal-col-bkg-lighter__deprecate)
+    );
+    width: 100%;
+    height: 2.5rem;
+    display: flex;
+    align-items: center;
+    column-gap: 0.5rem;
+    padding: 0 1rem;
+    cursor: pointer;
+
+    i {
+      font-size: 1rem;
+      width: 1.5rem;
+      height: 1.5rem;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+
+    .legend-container__header-icon {
+      color: var(--portal-col-primary-6);
+    }
+
+    .legend-container__header-expand {
+      color: var(--portal-col-primary-5);
+      display: none;
+    }
+
+    .legend-container__header-name {
+      flex-grow: 1;
+      color: var(--portal-col-primary-8);
+      font-size: 0.8rem;
+      font-weight: 500;
+    }
+  }
+
+  .legend-container__content {
+    padding-bottom: 0.5rem;
+    row-gap: 0.5rem;
+    flex-direction: column;
+    display: none;
+
+    .layer-legend {
+      padding: 0 1rem;
+
+      .layer-legend__layer-name {
+        padding: 0.25rem 0;
+        color: var(--portal-col-neutral-6);
+        font-size: 0.8rem;
+        font-weight: 500;
+      }
+
+      .layer-legend__palette {
+        color: var(--portal-col-neutral-8);
+        font-size: 0.8rem;
+        font-weight: 500;
+      }
+    }
+  }
+
+  &.expanded {
```

Comment:
I think typically in BEM it would be `legend-container--expanded` but not a big deal

Reply:
done!